README.md

```diff
@@ -15,7 +15,7 @@ The GOV.UK country and territory autocomplete  is a simple UI widget that allows
 
 The autocomplete uses data from the UK government's country and territory registers. To configure the data used in the autocomplete to suit the specific needs of your product or service, please [raise an issue](https://github.com/alphagov/govuk-country-and-territory-autocomplete/issues/new).
 
-The autocomplete itself follows the common look and feel of GOV.UK in line with the [design principles](https://www.gov.uk/design-principles).
+The autocomplete itself follows the common look and feel of GOV.UK in line with the [design principles](https://www.gov.uk/design-principles), and should work with [recommended browsers](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in).
```

Comment:
This is what the corresponding bit of the [accessible-autocomplete README](https://github.com/alphagov/accessible-autocomplete/blob/main/README.md) looks like:

> `accessible-autocomplete` is a JavaScript autocomplete built from the ground up to be accessible. The design goals are:
> 
> - **Accessibility:** Following WAI-ARIA best practices and testing with assistive technologies.
> - **User experience:** Supporting a wide variety of user needs.
> - **Compatibility:** Working with [recommended browsers](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in) and [assistive technologies](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies#which-assistive-technologies-to-test-with).
> - 

Should we also mention assistive tech in this PR? Also, should the text here be the same as in the accessible-autocomplete README?

Reply:
Re #23, I've raised a PR: (https://github.com/alphagov/govuk-country-and-territory-autocomplete/pull/101)

Currently, that PR does not contain any content about recommended browsers and assistive tech. I reckon making both READMEs align in every way possible is beyond the scope here. Would it be ok if we added the 2 links into this line, as below?

```suggestion
The autocomplete itself follows the common look and feel of GOV.UK in line with the [design principles](https://www.gov.uk/design-principles), and should work with [recommended browsers](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in) and [assistive technologies](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies#which-assistive-technologies-to-test-with).
```